tests/test_makepackage.py

```diff
@@ -1,57 +1,78 @@
-import platform
 import subprocess
+import platform
 from pathlib import Path
-from typing import Dict, List
+from typing import Dict, List, Tuple
```

Comment:
To be honest, with time, I am becoming more and more against hint typing test files. What's the point of using these type hints? All the arguments are fixtures and arguments from parametrizations. So, these type hints do not inform the users what types they should  use for arguments. Rather, they're informing what types the fixtures and arguments have. While this can have some minor sense, all in all this is not needed too much. Hence, I'd suggest getting rid of all type hints in the test files.

Now, what do you think? Maybe you know some counterarguments?

---

If we decide to go for type hints, I have a couple of comments. (But first let's discuss whether to use them at all.)

Please remove these imports. Instead, add as the first line of the module the following:
```
from __future__ import annotations
```
Now, you can make the following replacements:
* `Dict` → `dict`
* `List` → `list`
* `Tuple` → `tuple`